application/src/main/java/org/togetherjava/tjbot/commands/help/UserBannedDeleteRecentThreadsListener.java

```diff
@@ -42,12 +43,12 @@ public UserBannedDeleteRecentThreadsListener(Database database) {
 
     @Override
     public void onGuildBan(GuildBanEvent event) {
-        getRecentHelpThreads(event.getUser()).forEach(channelId -> event.getJDA()
-            .getThreadChannelById(channelId)
-            .delete()
-            .queue(onSuccess -> {
-            }, onFailure -> logger.warn("Failed to delete thread {} from banned user.", channelId,
-                    onFailure)));
+        getRecentHelpThreads(event.getUser()).stream()
+            .map(event.getJDA()::getThreadChannelById)
+            .filter(Objects::nonNull)
+            .forEach(threadChannel -> threadChannel.delete().queue(any -> {
+            }, failure -> logger.warn("Failed to delete thread {} from banned user.",
+                    threadChannel.getId(), failure)));
```

Comment:
might also want to log the id of the banned user

Reply:
that'll require me to change the whole thing again

Reply:
done